### PR TITLE
support command+enter on MacOS to submit post

### LINKED ISSUE #7273: Support command+enter on MacOS to submit post
Facebook and Twitter allow you to use ctrl+enter (on Macs are command+enter) keys to submit a post. It would be nice if Diaspora has that.

### PR DESCRIPTION
**Pull request by rajanak**
Fixes #7273

### CODE REVIEW

File: app/assets/javascripts/app/views/comment_stream_view.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ app.views.CommentStream = app.views.Base.extend({
   },
 
   keyDownOnCommentBox: function(evt) {
-    if(evt.which === Keycodes.ENTER && evt.ctrlKey) {
+    if(evt.which === Keycodes.ENTER && (evt.metaKey || evt.ctrlKey)) {
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/comment_stream_view.js]: Expected space(s) after "if".

File: app/assets/javascripts/app/views/conversations_form_view.js [hunk 2]
```diff
@@ -65,7 +65,7 @@ app.views.ConversationsForm = Backbone.View.extend({
   },
 
   keyDown: function(evt) {
-    if (evt.which === Keycodes.ENTER && evt.ctrlKey) {
+    if(evt.which === Keycodes.ENTER && (evt.metaKey || evt.ctrlKey)) {
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/conversations_form_view.js]: Expected space(s) after "if".

File: app/assets/javascripts/app/views/publisher_view.js [hunk 3]
```diff
@@ -352,7 +352,7 @@ app.views.Publisher = Backbone.View.extend({
   },
 
   keyDown : function(evt) {
-    if(evt.which === Keycodes.ENTER && evt.ctrlKey) {
+    if(evt.which === Keycodes.ENTER && (evt.metaKey || evt.ctrlKey)) {
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher_view.js]: Expected space(s) after "if".

**Review by SuperTux88 — CHANGES_REQUESTED** [review]
Thanks for the pull request. Looks good to me so far. 👍 

Can you please add some tests like we now have with the `ctrlKey` also with the `metaKey`? And then can you squash the commits to a single one?

**Comment by SuperTux88** [comment]
Superseded by #7524

**Pull request closed without merging**